Holodeck/Command.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from .Agents import *
+
+
+class Commands(object):
+    def __init__(self):
+        self._commands = []
+
+    def add_command(self, command):
+        self._commands.append(command)
+
+    def to_json(self):
```

Comment:
This could be a little simpler:
commands = ",".join(map(lambda x: x.to_json(), self._commands))
return "{\commands\": [" + commands + "]}"
Something like that. What do you think?

Reply:
That's much nicer. I'll implement it that way.